File: frontend/src/services/commentService.js
import {instance} from "../api/apiClient";

const commentService = {
    createComment: async (params) =>{
        return await instance.post("/comment", params);
    },
    deleteComment: async (commentId) =>{
        await instance.delete("/comment/"+commentId);
    },
};

export default commentService;

File: backend/src/main/java/com/study/backend/comment/service/CommentService.java
package com.study.backend.comment.service;

import java.util.List;

import com.study.backend.comment.model.Comment;

public interface CommentService {
	List<Comment> getCommentsByBoardId(Long boardId);
	Comment createComment(Comment comment);
	void deleteComment(Long commentId, Long memberId);
}
